src/working_files.cc

```diff
@@ -20,12 +20,12 @@ constexpr int kMaxDiff = 20;
 constexpr int kMaxColumnAlignSize = 200;
 
 lsPosition GetPositionForOffset(const std::string& content, int offset) {
-  if (offset >= content.size())
+  if (offset >= (int)content.size())
```

Comment:
Here's an example why it is dangerous to use unsigned types.. (size_t)0 - 1  blows up

Reply:
I know about the wrapping around of the unsigned and the undefined behaviour of signed integers. Apparently, I didn't pay enough attention to it.